### DISCUSSION TITLE
new syntax for looping n times without needing a variable

### DISCUSSION THREAD

**Discussion by BaddRadish** [godotengine/godot-proposals#5938] [Scripting]
suppose you want to do something n times, but dont need to track how many times you've done it.
currently the syntax is `for _n in n:` or maybe `for _n in range(n)` not sure.

how about `for n:`?

**Comment by dalexeev** [comment]
See #1282.

**Comment by Calinou** [comment]
This is being tracked in https://github.com/godotengine/godot-proposals/issues/1282.
